src/current/v23.2/read-committed.md

```diff
@@ -0,0 +1,936 @@
+
+	{% include_cached copy-clipboard.html %}
+	~~~ sql
+	BEGIN;
+	  SET transaction_isolation = 'read committed';
+	~~~
+
+To view the isolation level of a transaction, run `SHOW` within the open transaction:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SHOW transaction_isolation;
+~~~
+
+~~~
+  transaction_isolation
+-------------------------
+  read committed
+~~~
+
+{{site.data.alerts.callout_success}}
+Starting a transaction as `READ COMMITTED` does not affect the [session-level isolation setting](#set-the-current-session-to-read-committed), which can be different.
+{{site.data.alerts.end}}
+
+## `READ COMMITTED` transaction behavior
+
+`READ COMMITTED` and `SERIALIZABLE` transactions both serve globally consistent ("non-stale") reads and [commit atomically]({% link {{ page.version.version }}/developer-basics.md %}#how-transactions-work-in-cockroachdb). `READ COMMITTED` transactions have the following differences:
+
+- Writes in concurrent `READ COMMITTED` transactions can interleave without aborting transactions, and a write can never block a non-locking read of the same row. This is because `READ COMMITTED` transactions are not required to be placed into a [serializable ordering]({% link {{ page.version.version }}/demo-serializable.md %}). 
+
+- Whereas statements in `SERIALIZABLE` transactions see data that committed before the transaction began, statements in `READ COMMITTED` transactions see data that committed before each **statement** began. If rows are being updated by concurrent writes, reads in a `READ COMMITTED` transaction can [return different results](#non-repeatable-reads-and-phantom-reads).
+
+	{{site.data.alerts.callout_info}}
+	For details on how this is implemented, see [Read snapshots]({% link {{ page.version.version }}/architecture/transaction-layer.md %}#read-snapshots).
+	{{site.data.alerts.end}}
+
+- Due to the preceding behaviors, `READ COMMITTED` transactions permit some types of concurrency anomalies that are prevented in `SERIALIZABLE` transactions. For details and examples, see [Concurrency anomalies](#concurrency-anomalies).
+
+- You can mitigate concurrency anomalies by issuing [locking reads](#locking-reads) in `READ COMMITTED` transactions. These statements can block concurrent transactions that are issuing writes or other locking reads on the same rows.
+
+- When using `READ COMMITTED` isolation, you do not need to implement [client-side retries]({% link {{ page.version.version }}/transaction-retry-error-reference.md %}#client-side-retry-handling) to handle `40001` errors. This is because under [transaction contention]({% link {{ page.version.version }}/performance-best-practices-overview.md %}#transaction-contention), `READ COMMITTED` transactions will **not** return [serialization errors]({% link {{ page.version.version }}/transaction-retry-error-reference.md %}) to applications.
```

Comment:
Same comment as above: in light of @nvanbenschoten's findings in https://github.com/cockroachdb/cockroach/pull/117096 I think we probably need to cut out this paragraph, at least for 23.2. It seems like applications *will* still need client-side retry loops in 23.2 under RC isolation.

Reply:
I read the linked issue, but need help understanding how to contextualize it. It looks like serializable errors can still occur upon lease transfers. When will a customer be likely to encounter this, and will most or just a subset of customers encounter it? @michae2 @nvanbenschoten 

If it's safest, I can just remove the paragraph and tweak others that mention client-side retries.

Reply:
My understanding is that _all_ customers running without a client-side retry loop would encounter it after a few minutes of normal database activity. I think we should play it safe for 23.2 and remove this paragraph (with the hope that we'll be able to bring it back for 24.1! 🙂).

Reply:
@taroface I see this paragraph is still here? I think we need to remove it for 23.2.

Reply:
It's commented out using the liquid tags, so it won't show on the public site! Just verified in the preview as well.

Reply:
Oh, sorry! Thank you for setting me straight!